src/main/scala/no/ndla/learningpathapi/service/ConverterServiceComponent.scala

```diff
@@ -277,6 +265,7 @@ trait ConverterServiceComponent {
     def asLearningStepContainerSummary(status: StepStatus.Value, learningPath: domain.LearningPath, language: String): api.LearningStepContainerSummary = {
       val learningSteps = learningPathRepository.learningStepsFor(learningPath.id.get).filter(_.status == status)
       val supportedLanguages = learningSteps.flatMap(_.title).flatMap(_.language).distinct
+      LanguageValidator.checkIfLanguageIsSupported(supportedLanguages, language)
```

Comment:
Dersom lagnuage ikke finnes i supportedLanguages vil det kastes en `ValidationException` som fører til at APiet returnerer en 400 (bad request). I og med at vi har gått for 404 (not found) tror jeg det er bedre at vi holder det konsistent, og at vi retrunerer 404 her også. Dersom disse funksjonene returnerer en `Option` kan det returneres en `None` dersom en ressurs ikke finnes på språket, så håndtere dette i `ReadService` og controlleren